tests/testthat/test_stan_jm.R

```diff
@@ -520,6 +520,17 @@ for (j in c(1:30)) {
       }  
     }) 
     
+    test_that("posterior_traj works with new data with and without offset", {
+      nd <- pbcLong[1:2, ]
+      expect_s3_class(posterior_traj(f1, newdataLong = nd, dynamic = FALSE), "predict.stanjm")
+      
+      pbcLong2 <- pbcLong
+      pbcLong2$off <- 1
+      o <- SW(f1_off <- update(f1, dataLong = pbcLong2, formulaLong. = logBili ~ offset(off) + year + (year | id)))
+      nd <- pbcLong2[1:2, ]
```

Comment:
Do we need to drop the response column from the new data, to check that this test is passing when the new data doesn't contain the response (I think that was the original issue)?

We could extract just the `id`, `year` and `offset` columns I think, something like:

```
nd <- pbcLong2[1:2, c("id", "year", "off")]
```